lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -4,6 +4,13 @@ module LogStash; module Outputs; class ElasticSearch;
   module Common
     attr_reader :client, :hosts
 
+    # These are codes for temporary recoverable conditions
+    # 429 just means that ES has too much traffic ATM
+    # 503 means it , or a proxy is temporarily unavailable
+    RETRYABLE_CODES = [429, 503]
+
+    # These codes are not recoverable, send them to the DLQ
```

Comment:
It's somewhat out of this scope, but since the comment was added, the description for codes we put in DLQ_CODES doesn't seem consistent, as a 404 return code for an index operation may be fixed by creating the index?
if you agree I suggest leaving this without a comment and restrict the PR to fixing the missing constant + data loss